src/main/java/edu/wpi/grip/ui/Main.java

```diff
@@ -62,5 +65,11 @@ public final void onFatalErrorEvent(FatalErrorEvent error) {
                 }
             }
         });
+
+        if(event.isFatal()) {
+            System.err.println("Original fatal exception");
+            event.getThrowable().printStackTrace();
```

Comment:
I want to make it entirely clear what the original stack trace is just in case more come later.